### ISSUE TITLE
How to obtain the "Category_rev_ids" images?

### ISSUE THREAD

**Issue by zzw1123** [Gaoyiminggithub/Graphonomy#7]
Hi,
Thanks for your contribution!
I know that images in "Category_rev_ids" directory is obtained by flipping. But I also have to exchange the label of the body parts such as 'left-arm' and 'right-arm'? Am I right?

**Comment by Gaoyiminggithub** [comment]
Hi,
You can download the 'rev' images. The link is in the README.md.
The images downloaded from Readme.me in the 'rev' have been flipped which you don't need to exchange the label.

**Comment by zzw1123** [comment]
@Gaoyiminggithub 
I have ignored that. Thanks for your kind reply!

**Comment by zzw1123** [comment]
Sorry but I have another question......
In 'exp/transfer/train_cihp_from_pascal.py', there is a normalization applied to dataloader:
```
tr.Normalize_xception_tf(),
```
I do not know why you do this rather than standardizing them by: x-mean/std. Could you please tell me the difference between this two methods?
Thanks a lot!

**Comment by Gaoyiminggithub** [comment]
Hi, 
It is an interesting question.
Because the pretrained model is transferred from the [Deeplab](https://github.com/tensorflow/models/tree/master/research/deeplab) office code, the Deeplab code used the normalization way in [here](https://github.com/tensorflow/models/blob/master/research/deeplab/core/feature_extractor.py#L194-L197). So we use the same normalization way as it.

**Comment by Gaoyiminggithub** [comment]
Moreover, you can use the normalization way like x-mean/std to train the base model.

**Comment by zzw1123** [comment]
I think the normalization method makes little effect to the final performance, right?
